tests/unit/output/test_local.py

```diff
@@ -23,27 +23,27 @@ def test_save_missing(self):
                 o.save()
 
 
-def test_str_workdir_outside_repo(erepo):
-    stage = Stage(erepo.dvc)
+def test_str_workdir_outside_repo(erepo_dir):
+    stage = Stage(erepo_dir.dvc)
     output = OutputLOCAL(stage, "path", cache=False)
 
-    assert relpath("path", erepo.dvc.root_dir) == str(output)
+    assert relpath("path", erepo_dir.dvc.root_dir) == str(output)
```

Comment:
Btw first time noticing that it uses erepo 👀 Makes sense though...
```suggestion
    assert relpath("path", fspath(erepo_dir)) == str(output)
```